### PR TITLE
TE-4544 Updated all dependencies.

### PR DESCRIPTION
**Pull request by stereomon**
- Developer(s): @stereomon

- Ticket: https://spryker.atlassian.net/browse/TE-4544

- Academy PR: ACADEMY_URL_HERE

- Release Group: URL_HERE

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   Install               | patch (currently 0.x)  |                       |

#### Release Notes

{skip}

-----------------------------------------

##### Change log

Bugfixes

- Allow next major of Symfony.
- Allow next major of Monolog.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,18 +4,18 @@
...
+    "symfony/console": "^3.0.0 || ^4.0.0 || ^5.0.0",
+    "symfony/process": "^3.0.0 || ^4.0.0 || ^5.0.0",
+    "symfony/yaml": "^3.0.0 || ^4.0.0 || ^5.0.0",
+    "monolog/monolog": "^1.23 || ^2.0.0"
   },
   "require-dev": {
     "codeception/codeception": "*",
     "symfony/var-dumper": "*",
-    "phpstan/phpstan": "^0.11.2",
+    "phpstan/phpstan": "*",
```
Comment (dereuromark) [inline composer.json]: I wouldnt do this, phpstan is too bc breaking

**Comment by dereuromark** [comment]
One issue left?

>   Line   Spryker/Install/Executable/CommandLine/CommandLineExecutable.php  
 ------ ------------------------------------------------------------------ 
  70     Call to function method_exists() with 'Symfony\\Component…' and   
         'fromShellCommandline' will always evaluate to false.

File: .travis.yml [hunk 2]
```diff
@@ -22,9 +22,9 @@ before_script:
     - composer install --prefer-dist --no-interaction
 
 script:
-  - if [[ ${TRAVIS_PHP_VERSION} == "7.4" ]] ; then vendor/bin/codecept run  ; fi
-  - if [[ ${TRAVIS_PHP_VERSION} != "7.4" ]] ; then vendor/bin/codecept run --coverage-xml ; fi
-  - if [[ $CHECKS == 1 ]] ; then vendor/bin/phpstan analyse -l 7 -c phpstan.neon src/ ; fi
+  - if [[ ${TRAVIS_PHP_VERSION} == "7.4snapshot" ]] ; then vendor/bin/codecept run  ; fi
+  - if [[ ${TRAVIS_PHP_VERSION} != "7.4snapshot" ]] ; then vendor/bin/codecept run --coverage-xml ; fi
```
Comment (dereuromark) [inline .travis.yml]: 7.4 is fine now

File: composer.json [hunk 3]
```diff
@@ -4,18 +4,18 @@
...
+    "symfony/process": "^3.0.0 || ^4.0.0 || ^5.0.0",
+    "symfony/yaml": "^3.0.0 || ^4.0.0 || ^5.0.0",
+    "monolog/monolog": "^1.23 || ^2.0.0"
   },
   "require-dev": {
-    "codeception/codeception": "*",
+    "codeception/codeception": "^4.0.0",
     "symfony/var-dumper": "*",
-    "phpstan/phpstan": "^0.11.2",
+    "phpstan/phpstan": "*",
```
Comment (dereuromark) [inline composer.json]: 0.12 please

**Comment by dereuromark** [comment]
Do we have prefer lowest check activated for travis? Without this the promised majors here cannot be guaranteed.

**Comment by dereuromark** [comment]
As I suspected: https://travis-ci.org/spryker/install/jobs/633783009#L436

**Comment by dereuromark** [comment]
Do we want to activate Spryker CI on this one? is this now a module?
Currently it is still somewhat listed as "tool" only.

**Comment by stereomon** [comment]
This is still just a "tool".

**Pull request merged**